### DISCUSSION TITLE
The web3model css is faulty

### DISCUSSION THREAD

**Discussion by EV-Ain** [WalletConnect/walletconnect-monorepo#3947] [Developer Support]
![1700299422621](https://github.com/WalletConnect/walletconnect-monorepo/assets/101503420/25659068-af14-42c2-85d3-4af86d602570)
![Snipaste_2023-11-18_17-25-01](https://github.com/WalletConnect/walletconnect-monorepo/assets/101503420/3018fd8b-6839-4044-9784-85aa69fecfc8)
the reason is<html lang="en" style="font-size: 52px;"> how to deal with

**Comment by EV-Ain** [comment]
the reason is html fontsize:52px

**Comment by glitch-txs** [comment]
hi, could you share a public repository / code sandbox to reproduce the issue?

**Reply by EV-Ain** [reply]
https://github.com/EV-Ain/walletContent.git
